packages/shared/src/services/blockscout/UniversalBlockscoutClient.ts

```diff
@@ -3,10 +3,10 @@ import { ChainId, UnixTime } from '@l2beat/shared-pure'
 
 import { HttpClient } from '../HttpClient'
 import { BlockNumberProvider } from '../providers/BlockNumberProvider'
-import { RoutescanLikeClient } from '../routescan/RoutescanLikeClient'
+import { BlockscoutLikeClient } from './BlockscoutLikeClient'
 
-export class UniversalRoutescanClient
-  extends RoutescanLikeClient
+export class UniversalBlockscoutClient
```

Comment:
I really dislike having both `UniversalBlockscoutClient` and `BlockscoutLikeClient`. Same for Etherscan. IMO passing chainId to them was simply a mistake. There should only be one class - the universal one.